deposit/static/deposit/js/overview.js

```diff
@@ -59,29 +112,13 @@ async function handleFolderDelete (collection, folderName) {
 
   if (result.status === 'ok') {
     Yoda.set_message('success', 'Successfully deleted deposit ' + folderName)
-    browse(collection, true)
+    buildFileBrowser()
     $('#deposit-delete').modal('hide')
   } else {
     fileMgmtDialogAlert('deposit-delete', result.status_info)
   }
 }
 
-function changeBrowserUrl (path) {
```

Comment:
It didn't seem like we were using this function, so I removed it, but is there a way we use it in deposit?